README.md

```diff
@@ -93,6 +93,33 @@ pip install -r requirements.txt
 ./whoogle-search
 ```
 
+#### systemd Configuration
+After building the virtual environment, add the following to `/lib/systemd/system/whoogle.service`:
```

Comment:
```suggestion
After building the virtual environment, you can add the following to `/lib/systemd/system/whoogle.service` to set up a Whoogle Search systemd service:
```

Super minor nitpick here, but just for anyone who is unfamiliar with systemd, I don't want them confused into thinking that this is a required step after the manual setup steps above. Probably won't happen, but doesn't hurt to just make the wording a bit more clear.